### ISSUE TITLE
regionContainerType is set to SectionWithoutHeader

### ISSUE THREAD

**Issue by olvina** [forcedotcom/cli#1872]
### Summary

We have created a new released package version, but when we try to install it, it failed with

`(Screen Component) - Because regionContainerType is set to SectionWithoutHeader for the "name_of_section" section screen field on the "name_of_screen" screen, the fieldText field must be null.
`

We tried downgrading the API version of the flow in the scratch org to 52, 53, 54 and even upgrading it to 56, but the same error still displays when we try to generate a new package version: 

`sfdx force:package:beta:version:create --path force-app --loglevel debug --branch refs/pull/39/merge --installationkeybypass --targetdevhubusername name@domain.com --codecoverage --wait 60 --json`

We have checked the flow metadata and a reference to regionContainerType or SectionWithoutHeader is not found in the XML nor was it found in our entire project.

### Expected result

package version is created and able to be installed.

### System Information

- Which shell/terminal are you using? 
 zsh

sfdx: 
{
  "cliVersion": "sfdx-cli/7.182.1",
  "architecture": "darwin-x64",
  "nodeVersion": "node-v18.12.1",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 1.3.6 (core)",
    "@oclif/plugin-commands 2.2.1 (core)",
    "@oclif/plugin-help 5.1.19 (core)",
    "@oclif/plugin-not-found 2.3.9 (core)",
    "@oclif/plugin-plugins 2.1.8 (core)",
    "@oclif/plugin-search 0.0.4 (core)",
    "@oclif/plugin-update 3.0.7 (core)",
    "@oclif/plugin-version 1.1.3 (core)",
    "@oclif/plugin-warn-if-update-available 2.0.15 (core)",
    "@oclif/plugin-which 2.2.2 (core)",
    "alias 2.1.12 (core)",
    "apex 1.3.0 (core)",
    "auth 2.3.3 (core)",
    "community 2.0.18 (core)",
    "config 1.4.22 (core)",
    "custom-metadata 2.0.5 (core)",
    "data 2.1.17 (core)",
    "generator 2.0.13 (core)",
    "info 2.2.13 (core)",
    "limits 2.1.3 (core)",
    "org 2.2.19 (core)",
    "packaging 1.10.10 (core)",
    "schema 2.1.19 (core)",
    "signups 1.2.12 (core)",
    "source 2.3.8 (core)",
    "telemetry 2.0.5 (core)",
    "templates 55.1.0 (core)",
    "trust 2.2.2 (core)",
    "user 2.1.22 (core)",
    "@salesforce/sfdx-plugin-lwc-test 1.0.1 (core)",
    "@salesforce/sfdx-scanner 3.8.0 (user)",
    "salesforce-alm 54.8.4 (core)"
  ],
  "osVersion": "Darwin 21.3.0",
  "shell": "zsh",
  "rootPath": "/Users/username/.local/share/sfdx/client/7.182.1-dad7ae3"
}

**Comment by cristiand391** [comment]
Hey @olvina, can you reproduce this using the current (non-beta commands)?

`sfdx force:package:version:create --path force-app --loglevel debug --branch refs/pull/39/merge --installationkeybypass --targetdevhubusername name@domain.com --codecoverage --wait 60 --json`

**Comment by olvina** [comment]
Hi @cristiand391 we tried running the command you suggested. The error message was the same. We have even deleted the flow from our project and scratch org and the error is still displaying even when the flow no longer exists. This is puzzling

**Comment by cristiand391** [comment]
Flow docs:
https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_visual_workflow.htm

```
regionContainerType 

FlowRegionContainerType (enumeration of type string)

Stores information about a section component header. Possible values include:SectionWithHeaderSectionWithoutHeaderAvailable only when the component type is Section. Available in API version 55.0 and later.
```

So this should only work with API version >= 55.0.

Do you see any reference to this field like this one:
https://github.com/trailheadapps/functions-recipes/pull/480/files

After deleting the flow from your local project, try creating the pkg on a new scratch org.
